### DISCUSSION TITLE
Possible to import conversation from chatGPT to LibreChat?

### DISCUSSION THREAD

**Discussion by balajeek** [danny-avila/LibreChat#1283] [Q&A]
Tried to search for this in discussion and couldn't find one.

I have the conversations exported from chatGPT and i just got them to download the conversations.json file.  wondering if i could import them into my LibreChat instance, I thought the Import under the top toolbar will do, I did import and it showed 'undefined' list in them and when i click it says 'preset Active' (may be its not ment for that).

**Comment by fuegovic** [comment]
> when i click it says 'preset Active' ...

Indeed, the json import is for presets. Presets are similar to ChatGPT's instructions. There are no built in feature to import the conversations from ChatGPT

**Comment by danny-avila** [comment]
Hi @balajeek,

This is not currently a feature, thanks for opening a discussion on it. It will be on the roadmap but I'm not sure it's a big priority right now.

**Comment by balajeek** [comment]
Good to know, any way i can work with json file to include into certain folder structure to place it myself, just curious.

I do have habit to refer back my earlier conversations on chatGPT, if i am going to be using Libre,I hate to leave all those in chatGPT.
Thanks.

**Reply by fuegovic** [reply]
you'd have to look at the database structure. If your database is on docker you can use mongo-express to easly access it: [Manage Your MongoDB Database with Mongo Express](https://docs.librechat.ai/features/manage_your_database.html)

if you're using an online database you can use mongodb compass

**Reply by balajeek** [reply]
okay thanks, i will look into it to see if i can import it.

**Reply by dacog** [reply]
Hi @balajeek 

Did you manage to import chatGPT conversations into libre chat somehow? I've also interest in doing that as I also constantly refer to old conversation in chatGPT.

**Comment by danny-avila** [comment]
It's now possible to import conversations from ChatGPT, LibreChat, and Chatbot UI: https://github.com/danny-avila/LibreChat/pull/2355
